metricbeat/module/docker/diskio/helper.go

```diff
@@ -174,5 +175,12 @@ func calculatePerSecond(duration time.Duration, old uint64, new uint64) float64
 	if value < 0 {
 		value = 0
 	}
-	return value / duration.Seconds()
+
+	timeSec := duration.Seconds()
+	if timeSec == 0 {
+		logp.Err("duration is zero when calculating per-second values")
```

Comment:
If this happens continuously in some case this is going to flood logs. I think we could just `return 0` in this case.